environment.yml

```diff
@@ -42,6 +42,5 @@ dependencies:
   - xz
   - zlib
   - zstandard
-  - pip:
-    - eventio~=1.0
-    - https://github.com/cta-observatory/ctapipe-extra/archive/v0.3.0.tar.gz
+  - eventio>=1.0
```

Comment:
we require at least  1.1.1 for the calibration event type. I thought I updated this also here (in setup.py it's correct)

Reply:
change to `>=1.1.1,<2`

Reply:
done.  For the conda package, all specs are taken from setup.py, so if a user installs it via conda, it will be all ok.